clang/lib/AST/StmtPrinter.cpp

```diff
@@ -2556,6 +2558,15 @@ void StmtPrinter::VisitPackIndexingExpr(PackIndexingExpr *E) {
   OS << "]";
 }
 
+void StmtPrinter::VisitResolvedUnexpandedPackExpr(
+    ResolvedUnexpandedPackExpr *E) {
+  OS << "<<resolved pack(";
+  llvm::interleave(
+      E->getExprs(), E->getExprs() + E->getNumExprs(),
```

Comment:
IMO, we should have `ResolvedUnexpandedPackExpr` return an `ArrayRef` instead, having to do these additions of the size everywhere is quite unfortunate.